lib/pbench/test/functional/server/test_user.py

```diff
@@ -80,7 +82,8 @@ def test_profile_noauth(self, pbench_server_client: PbenchServerClient):
         Test that we can't access a user profile without authentication.
         """
         with pytest.raises(
-            HTTPError, match=r"UNAUTHORIZED for url: http://.*?/api/v1/user/functional"
+            HTTPError,
+            match=f"UNAUTHORIZED for {pbench_server_client.scheme}: http://.*?/api/v1/user/{USERNAME1}",
```

Comment:
s/{pbench_server_client.scheme}/url/
s/http/{pbench_server_client.scheme}/
🙂 

(Um, how is this test passing?  🤯)

Reply:
I strongly suspect that I forgot to run the functional tests after the last round of changes, and they're not run automatically yet. Oops!